tests/Imagine/Filter/Loader/DownscaleFilterLoaderTest.php

```diff
@@ -61,8 +62,10 @@ public function provideSizes()
             ->willReturn($initialSize);
         $image
             ->method('resize')
-            ->willReturnCallback(function ($box) use (&$resultSize) {
+            ->willReturnCallback(function ($box) use ($image, &$resultSize): ImageInterface {
```

Comment:
should we add a type declaration for $box as well?

Reply:
sure, added it, I may have missed some. 

My idea was when we finishing adding type declarations, check the missing ones with phpstan